mmdet/datasets/coco.py

```diff
@@ -405,9 +405,6 @@ def evaluate_det_segm(self,
                 'bbox', 'segm', 'proposal', 'proposal_fast'.
             logger (logging.Logger | str | None): Logger used for printing
                 related information during evaluation. Default: None.
-            jsonfile_prefix (str | None): The prefix of json files. It includes
-                the file path and the prefix of filename, e.g., "a/b/prefix".
-                If not specified, a temp file will be created. Default: None.
```

Comment:
There might be an extra delete happening?
Where did the changes to the `coco.py`?

Reply:
`jsonfile_prefix` is not a arg of `evaluate_det_segm`, was forgot to be deleted in last PR.